src/components/ADempiere/Form/VPOS/Order/index.vue

```diff
@@ -52,6 +52,20 @@
                 :is-disabled="isDisabled"
               />
             </el-col>
+            <el-col :span="5" :style="styleTab">
+              <el-form-item>
```

Comment:
the `el-form-item` tag may be unnecessary since it is present inside the field component